future-apps/payroll/contracts/Payroll.sol

```diff
@@ -594,61 +561,68 @@ contract Payroll is AragonApp { //, IForwarder { // makes coverage crash (remove
                 0,
                 0,
                 1,
-                ""
+                "Payroll"
             );
             emit SendPayroll(employee.accountAddress, token, tokenAmount);
             somethingPaid = true;
         }
 
-        // try to remove employee
-        if (employees[employeeId].terminated &&
-            employees[employeeId].endDate <= getTimestamp64() &&
-            employees[employeeId].accruedValue == 0
+        // Try to remove employee
+        if (employee.terminated &&
+            employee.endDate <= timestamp &&
+            employee.accruedValue == 0
         ) {
-            delete employeeIds[employees[employeeId].accountAddress];
-            delete employees[employeeId];
+            delete employeeIds[employee.accountAddress];
+            delete employees[_employeeId];
         }
     }
 
-    function _terminateEmployee(uint128 employeeId, uint64 endDate) internal {
-        // prevent past termination dates
-        require(endDate >= getTimestamp64());
```

Comment:
See [this comment](https://github.com/aragon/aragon-apps/pull/394#discussion_r201396637). Pinging @izqui